arduino/smartcar_mqtt/smartcar_mqtt.ino

```diff
@@ -1,5 +1,8 @@
 #include <Smartcar.h>
 #include <MQTT.h>
+#ifdef __SMCE__
+#include <OV767X.h>
+#endif
```

Comment:
This is my "issue".
This is a **very** idealistic camera but to be honest, it's how a camera API should be. I have used the particular library (or rather one for the specific sensors) and  to grab a frame you need to write a lot and ugly code from what I remember. :cry: 

Anyway, I don't have time to try it right now, but I suggest we include an "SMCE camera class" in the smartcar shield library instead, available only for the emulator.